scanomatic/data_processing/pheno/save.py

```diff
@@ -257,8 +263,8 @@ def zipit(save_functions, data, zip_paths):
     zip_paths.append(
         os.path.join(dir_path, _paths.phenotypes_extraction_params),
     )
-    save_functions.append(np.save)
-    data.append(settings.serialized())
+    save_functions.append(lambda fh, obj: json.dump(obj, fh))
+    data.append(json.loads(jsonizer.dumps(settings.asdict())))
```

Comment:
Should be something like this
```suggestion
    save_functions.append(lambda fh, obj: fh.write(jsonizer.dumps(obj)))
    data.append(settings)
```

Reply:
I guess the `SOMSerializers` and `SOMEncoder` needs to be expanded for that to work..?

Reply:
yes SOMEncoder only needs an `isinstance(o, dataclass)` and then `asdict(o)` in it

Reply:
and the `decode_dataclass` should be fairly straight forward. Just a `lookup[name](**obj[CONTENT])`

Reply:
fixed